spring-cloud-gcp-autoconfigure/src/main/java/com/google/cloud/spring/autoconfigure/sql/GcpCloudSqlProperties.java

```diff
@@ -1,74 +0,0 @@
-/*
- * Copyright 2017-2020 the original author or authors.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      https://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.google.cloud.spring.autoconfigure.sql;
-
-import com.google.cloud.spring.core.Credentials;
-
-import org.springframework.boot.context.properties.ConfigurationProperties;
-
-/**
- * Google Cloud SQL properties.
- *
- * @author João André Martins
- * @author Øystein Urdahl Hardeng
- */
-@ConfigurationProperties("spring.cloud.gcp.sql")
-public class GcpCloudSqlProperties {
```

Comment:
I am confused. If you remove this class, no metadata is going to be generated for those keys anymore. If you hare exclusively using those in the `EnvironmentPostProcessor` you could keep this object and binds it programmatically using the `Binder`. If you don't, you need to generate the metadata manually so that those keys are still advertized in the IDE.

Reply:
I didn't know about `Binder` and forgot about the metadata support. I'll look into it.

Reply:
Can you please see if I did it right in the last commit? Thanks!

Reply:
Thanks for the follow-up. The use of `Binder` looks correct to me.